scripts/localization.ttslua

```diff
@@ -458,6 +464,597 @@ LOCALIZED_STRINGS = {
+    --["Robber"] = "Struikrover", --follower on a road - Shared with Robber name from The Robber
+    ["Monk"] = "Monnik", --follower in a cloister
+    --Neutral Figure Names
+    ["Dragon"] = "Draak",
+    ["Fairy"] = "Fee",
+    ["Count"] = "Graaf",
+    ["Tower"] = "Toren",
+    ["Towers"] = "Torens",
+    ["Bridge"] = "Brug",
+    ["Bridges"] = "Bruggen",
+    ["Castle"] = "Kasteel",
+    ["Castles"] = "Kastelen",
+    ["Gold Bar"] = "Goudstaaf",
+    --Feature Names
+    ["Field"] = "Weide",
+    ["Fields"] = "Weides",
+    ["City"] = "Stad",
+    ["Cities"] = "Steden",
+    ["Road"] = "Weg",
+    ["Roads"] = "Wegen",
+    ["Cloister"] = "Klooster",
+    ["Cloisters"] = "Kloosters",
+    --Situational Feature Names
+    ["Farm"] = "Boerderij", --field with a follower on it
+    ["Farms"] = "Boerderijen", --field with a follower on it
+    --Special Feature Names
+    ["Cathedral"] = "Kathedraal",
+    ["Cathedrals"] = "Kathedralen",
+    ["Inn"] = "Herberg",
+    ["Inns"] = "Herbergen",
+    ["Princess"] = "Jonkvrouw",
+    ["Volcano"] = "Vulkaan",
+    ["Portal"] = "Poort", --from big box 3 manual (bad translation?)
+    ["Portals"] = "Poorten",
+    ["Tower Foundation"] = "Torenbouwplaats",
+    ["Abbey"] = "Abdij",
+    ["Shrine"] = "Cultusplaats",
+    ["Shrines"] = "Cultusplaatsen",
+    ["Hill"] = "Heuvel",
+    ["Hills"] = "Heuvels",
+    ["Vineyard"] = "Wijngaard",
+    ["Vineyards"] = "Wijngaarden",
+    ["Bazaar"] = "Bazaar",
+    ["Bazaars"] = "Bazaars",
+    ["Fliers"] = "Vliegtuigen",
+    ["Messages"] = "Nieuwsberichten",
+    ["Ferries"] = "Veerboten",
+    ["Gold"] = "Goud", --Should the special feature just be called gold bars?
+    --City of Carcassonne Quarters
+    ["Castle Quarter"] = "Kasteel",
+    ["Blacksmith Quarter"] = "Smederij",
+    ["Cathedral Quarter"] = "Kathedraal",
+    ["Market Quarter"] = "Markt",
+    --Wheel of Fortune Features
+    ["Crown"] = "Kroon",
+    ["Fortune"] = "Fortuin",
+    ["Taxes"] = "Belasting",
+    ["Famine"] = "Hongersnood",
+    ["Storm"] = "Storm",
+    ["Inquisition"] = "Inquisitie",
+    ["Plague"] = "Pest",
+    --["Dragon"] = "火龍", --shared with figure name
+    --Token Names
+    ["Trade Goods"] = "Handelswaren",
+    ["Wine Token"] = "Wijnfiche",
+    ["Cloth Token"] = "Lakenfiche",
+    ["Wheat Token"] = "Graanfiche",
+    --currently excluding the token name for display in the control panel
+    ["King"] = "Koning",
+    ["Robber Baron"] = "Rover", --currently shared with follower on a road. Consider renaming to Robber Baron later and possibly having a unique translation
+    ["Knock Out"] = "Taartgevecht",
+    ["Seduction"] = "Verliefd",
+    ["Target"] = "Schietwedstrijd",
+    ["Catch"] = "Vangen",
+    --Starting Tiles
+    ["Starting Tiles"] = "Starttegels",
+    ["Base Starting Tile"] = "Standaard starttegel",
+    ["The River"] = "De Rivier",
+    ["The River II"] = "De Rivier II",
+    ["The River (Big Box 5)"] = "De Rivier (Big Box 3)",

```

Comment:
This says Big Box 3, which I assume is a typo unless for some reason the dutch version of big box 5 had different numbering :)

Reply:
No typo. The Dutch version of Carcassonne has only three Big Box releases and as far as I can tell, our Big Box 3 comes closest to the content of what is elsewhere called Big Box 5.

Reply:
Oh interesting. I stand corrected!